CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+Contributing to WildFly Elytron
+======================
+
+- [Contributing to WildFly Elytron](#contributing-to-wildfly-elytron)
+  * [Pre-requisites](#pre-requisites)
+  * [Issues](#issues)
+    + [Good First Issues](#good-first-issues)
+  * [Setup Development Environment](#setup-development-environment)
+  * [Coding Guidlines](#coding-guidlines)
+
+
+## Pre-requisites 
+To contribute, you will first need to fork the [wildfly-elytron](https://github.com/wildfly-security/wildfly-elytron) repository. 
+
+This can be done by looking in the top-right corner of the repository page and click "Fork".
+![fork](https://docs.github.com/assets/images/help/repository/fork_button.jpg)
+
+The next step is to clone your newly forked repository onto your local workspace. This can be done by going to your newly forked repository, which should be at `https://github.com/USERNAME/wildfly-elytron`. 
+
+Then there will be a green button that says "Code", select that then copy the url.
```

Comment:
Then, there will be a green button that says "Code". Click on that and copy the URL.

Reply:
Changed